### ISSUE TITLE
I want to toggle the option for a match while it's running

### ISSUE THREAD

**Issue by yuki-ycino** [junegunn/fzf#2245]
- [x] I have read through the manual page (`man fzf`)
- [x] I have the latest version of fzf
- [x] I have searched through the existing issues

## Info

- OS
    - [ ] Linux
    - [x] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Shell
    - [ ] bash
    - [x] zsh
    - [ ] fish
    
## Problem / Steps to reproduce

I'd like to switch the options for matches in the middle of running fzf when doing a file search, etc.
Is it possible to implement the ability to bind the key to `--exact` or `--no-sort` switching?

**Comment by junegunn** [comment]
There is `toggle-sort` action you can use with `--bind`.

https://github.com/junegunn/fzf/blob/722d66e85abde02518214edd1ab186d321c0170c/shell/key-bindings.bash#L52

I didn't implement an action for toggling "exactness", because
1. we need to clear caches here and there to switch the search mode so it's a bit more complicated
2. exact matches will still be the top matches even in the fuzzy mode
3. we can prepend `'` to flip exactness term by term. e.g. `bar 'foo`

**Comment by yuki-ycino** [comment]
Thanks.
I missed that about toggle-sort.

I am making a vim plugin that uses fzf.
https://github.com/yuki-ycino/fzf-preview.vim

I'm implementing MRU (Most Recently Used) in that plugin, and in order to make the order useful, I need the --no-sort option.

While fuzzy mode is useful in many cases, it is sometimes easier to use --no-sort and --exact, for example for MRU.
So I'd like to allow the user to toggle both options at the same time.

Can you come up with a good solution for this?

**Comment by junegunn** [comment]
I'm not sure why you have to toggle both options at the same time.

Your users can already put `'` prefix on each term for toggling exact/fuzziness, which gives them more control.

https://github.com/junegunn/fzf#search-syntax

**Comment by yuki-ycino** [comment]
I know that you can make it exact with single quote prefixes.
I tried it myself for a while and found that switching options at the same time was more intuitive for the plugin.
So I was thinking of implementing a simultaneous switching operation if possible.

However, if it doesn't seem to be a good feature for fzf, I'm going to give it up.
Thank you.

**Comment by kevinhwang91** [comment]
+1

If the most case is wanna exact-match, the toggle option is convenient for the edge cases.

**Comment by junegunn** [comment]
> If the most case is wanna exact-match

Then you can start with `--exact` and "un-exact" each term with `'` prefix. e.g. `exact-term 'fuzzy-term`

> awkward when using c-u or c-w

FYI, `alt-backspace`, `alt-b`, `alt-f`, and such are great in such scenarios and I use them all the time. They are standard Readline bindings that you can use everywhere.

**Comment by kevinhwang91** [comment]
> > If the most case is wanna exact-match
> 
> Then you can start with `--exact` and "un-exact" each term with `'` prefix. e.g. `exact-term 'fuzzy-term`
> 
> > awkward when using c-u or c-w
> 
> FYI, `alt-backspace`, `alt-b`, `alt-f`, and such are great in such scenarios and I use them all the time. They are standard Readline bindings that you can use everywhere.

Yes, this workaround solves my need, forgive me for misunderstanding `unquotes` in doc.
>   Exact-match by default
       If you don't prefer fuzzy matching and do not wish to "quote" (prefixing with ') every word, start fzf with -e
       or --exact option. Note that when --exact is set, '-prefix "unquotes" the term.

**Comment by junegunn** [comment]
No problem. I guess the sentence needs some improvement.
